### PR TITLE
Updated run.sh to validate postsubmit jobs

### PR DESCRIPTION
**Pull request by yashsingh74**

### CODE REVIEW

File: images/test-image/run.sh [hunk 1]
```diff
@@ -10,4 +10,4 @@ command="${@:3}"
 
 git clone "https://github.com/${github_org}/${github_repo}"
 cd "${github_repo}"
-eval "${command}" || { cleanup_dind ; exit 1; }
+eval "${command}" || { cleanup_dind ; exit 1; }
```
Comment (rajaskakodkar) [inline images/test-image/run.sh]: I think we should add `cleanup_dind` after this as well to stop docker if ${command} returns 0

**Review by rajaskakodkar — CHANGES_REQUESTED** [review]

File: images/test-image/run.sh (hunk 1 above)
Comment (yashsingh74) [reply images/test-image/run.sh]: Done

**Comment by navidshaikh** [comment]
nit: to use the commit messages / PR title inline of the changes.

**Pull request merged**